.github/workflows/release.yml

```diff
@@ -47,5 +47,3 @@ jobs:
         dart pub global run flutter_plugin_tools publish-plugin --all-changed --base-sha=HEAD~ --skip-confirmation --remote=origin
       env: {PUB_CREDENTIALS: "${{ secrets.PUB_CREDENTIALS }}"}
 
-    env:
-      DEFAULT_BRANCH: master
```

Comment:
For posterity in case I'm wrong: we're removing this because AFAICT nothing uses it. I strongly suspect it's cruft from an earlier draft of this script.